src/shared/browser/chrome.ts

```diff
@@ -0,0 +1,65 @@
+import { IBrowser } from './types'
+
+export const chromeBrowser: IBrowser = {
+  storage: {
+    local: {
+      get: (key, defaultValue) => {
+        return new Promise((resolve) => {
+          chrome.storage.local.get([key], (storage) => {
```

Comment:
I think we can simplify this by retrieving data by key instead of [key] and object iteration later